packages/react-pose/src/components/PoseElement/index.tsx

```diff
@@ -167,33 +163,6 @@ class PoseElement extends React.PureComponent<PoseElementInternalProps> {
     return props;
   }
 
-  /**
-   * We need to get a ref to the underlying DOM element. Styled Components and
-   * other libraries use `innerRef`, though this will be swallowed if the
-   * styled component is not a primitive (ie styled(Component)).
-   *
-   * Instead we pass another function, `hostRef`, as recommended by Facebook
-   * https://github.com/facebook/react/issues/11401
-   *
-   * We also only pass `ref` to DOM primitive components.
-   */
-  getRefs = (): RefSetters => {
-    const refs: RefSetters = {};
-    const { elementType } = this.props;
-
-    if (
-      typeof elementType === 'string' ||
-      isForwardRef(elementType)
-    ) {
-      refs.ref = this.setRef;
-    } else {
-      refs.innerRef = this.setRef;
-      refs.hostRef = this.setRef;
-    }
-
-    return refs;
-  };
-
   setRef = (ref: Element) => {
```

Comment:
this has wrong type - should be `Element | null`, I also think that this method body can be unwrapped from this if statement in it, it really shouldnt matter, u either have that function called with `ref` on mount and null on unmount

additionally to provide helpful warnings for the users 2 warnings could get implemented:
- 1 here, validating that `setRef` is not called when `this.ref !== null` (or undefined, not sure whats it initial value)
- second one in `componentDidUpdate` - validating that `this.ref` is not null, this shouldnt happen because we cache the connected element, and its not possible to update it or disconnect it (we'll hold a reference to it anyway in poser)